app/controllers/posts_controller.rb

```diff
@@ -7,7 +7,7 @@ def index
 
 def show
   @posts = Post.all.sort_by { |post| post.created_at }.reverse
```

Comment:
Look into to doing this with `order`, so it does it in the database rather than Ruby (and is faster as a result).